Userland/Libraries/LibC/arch/x86_64/crti.S

```diff
@@ -1,15 +1,15 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
```

Comment:
That doesn't seem quite right ^^ (same with the headers for `crti.S` and `crtn.S`)